bedrock/firefox/templates/firefox/privacy/index.html

```diff
@@ -88,65 +62,61 @@ <h3 class="privacy-promise-sub-title">{{ ftl('firefox-privacy-hub-we-do-the-hard
           {{ ftl('firefox-privacy-hub-data-security-is-complicated') }}
         {% endif %}
       </p>
-    {% endcall %}
+    </div>
+  </div>
+
+  <div>
+    <h2>{{ ftl('firefox-privacy-hub-no-secrets') }}</h2>
+    <h4 class="privacy-promise-sub-title">{{ ftl('firefox-privacy-hub-youll-always-know-where-you') }}</h4>
+    <p>
+      {% if ftl_has_messages('firefox-privacy-hub-theres-no-hidden-agenda-here-v2') %}
+        {{ ftl('firefox-privacy-hub-theres-no-hidden-agenda-here-v2', privacy=url('privacy.notices.firefox')) }}
+      {% else %}
+        {{ ftl('firefox-privacy-hub-theres-no-hidden-agenda-here',
```

Comment:
**Suggestion:** You can remove the l10n string `firefox-privacy-hub-theres-no-hidden-agenda-here` here and in the .ftl file and just use the v2 one.

It's not blocking but we do like to clean up the obsolete strings when we touch a file that references them.